source/_patterns/03-organisms/sections/hero-cards/_hero-cards.scss

```diff
@@ -30,6 +30,9 @@
   }
 
   .jcc-cards-section {
+    margin-bottom: -1.5rem;
```

Comment:
Let's get rid of this negative margin. On the current homepage, there is an empty `.jcc-text-section` being incorrectly rendered. Let's fix that in the Drupal twig template.

Reply:
See https://github.com/chapter-three/jcc-srl/pull/326